samples/capture-to-file/encode-worker.js

```diff
@@ -0,0 +1,86 @@
+importScripts('./webm-writer2.js')
+
+let webmWriter = null;
+let fileWritableStream = null;
+let frameReader = null;
+
+async function startRecording(fileHandle, frameStream, trackSettings) {
+  let frameCounter = 0;
+
+  fileWritableStream = await fileHandle.createWritable();
+
+  webmWriter = new WebMWriter({
+      fileWriter: fileWritableStream,
+      codec: 'VP9',
+      width: trackSettings.width,
+      height: trackSettings.height});
+
+  frameReader = frameStream.getReader();
+
+  const init = {
+    output: (chunk) => {
+      webmWriter.addFrame(chunk);
+    },
+    error: (e) => {
+      console.log(e.message);
+      vtr.stop();
```

Comment:
Remove `vtr.stop()` :)

Is there extra cleanup needed for the `frameStream` or `frameReader`?